treescript/data/treescript_task_schema.json

```diff
@@ -59,7 +59,10 @@
                     "type": "boolean"
                 }
             }
-        }
+        },
+       "dontbuild": {
```

Comment:
This should be added as part of the payload (so in a nesting level, alongside "dry_run" and those properties)